test/e2e/framework/util.go

```diff
@@ -4576,6 +4576,122 @@ func CheckConnectivityToHost(f *Framework, nodeName, podName, host string, pingC
 	return err
 }
 
+// Regression test for #74839
+// https://github.com/kubernetes/kubernetes/issues/74839
+func CheckConnectionReset(f *Framework) error {
```

Comment:
This doesn't seem like a common enough thing to put in framework/util. Maybe put these back to the actual test file?

Reply:
Also would be great to have a high level description on what this test does --- as it seems to be different than what we usually test?